### ISSUE TITLE
Modify update_release_version to allow multiple TF versions during CI test

### ISSUE THREAD

**Issue by seanpmorgan** [tensorflow/addons#2258]
Going forward we'll want to test against multiple (maybe just oldest and newest supported versions) as shown below:
![image](https://user-images.githubusercontent.com/18154355/100569878-4040ea00-3284-11eb-9cba-29a58e42607c.png)
https://github.com/tensorflow/addons/pull/2250/files

This means we'll want to modify https://github.com/tensorflow/addons/blob/master/tools/update_release_version.sh to support that. Open to ideas on doing that... I believe just modifying the last version in that array should work.

**Comment by bhack** [comment]
> Going forward we'll want to test against multiple (maybe just oldest and newest supported versions) as shown below:
> ![image](https://user-images.githubusercontent.com/18154355/100569878-4040ea00-3284-11eb-9cba-29a58e42607c.png)
> https://github.com/tensorflow/addons/pull/2250/files
> 
> This means we'll want to modify https://github.com/tensorflow/addons/blob/master/tools/update_release_version.sh to support that. Open to ideas on doing that... I believe just modifying the last version in that array should work.

I don't but we could use `last_version` variables?

**Comment by seanpmorgan** [comment]
Yeah we should utilize a Min/Max TF variable where possible, but will still need the regex script for documentation

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
